icinga2/utils.py

```diff
@@ -184,7 +184,7 @@ def process_dict(attrs, indent=1, level=3, prefix=None):
 		if prefix is None:
 			prefix = "\t" * indent
 
-		for attr, value in attrs.items():
+		for attr, value in list(attrs.items()):
```

Comment:
Does it fail if we just iterate over the generator/dict_keys that gets returned?